docs/user-guide/04-modules-and-access-control.md

```diff
@@ -195,6 +191,39 @@ The Slang compiler enforces the following rules regarding access control:
 - Type definitions themselves cannot be `private`, for example, `private struct S {}` is not valid code.
 - `interface` requirements cannot be `private`.
 
+## Organizing Systems of Modules
+
+Slang does not seek to impose any specific organization of modules. However, there are some conventions that have emerged as being useful.
+
+### Module Organization Suggestions
+
+- Top-level modules are those that are `import`ed by user code.
```

Comment:
The terminology isn't very accurate: the `util.slang` is the "main module file", and files being `__included` are just files in that module. They are not "submodules".

Think about a module as a Visual Studio C++/C# Project, and a file is just a file in that project.